lab-bundle-spec.md

```diff
@@ -317,13 +322,46 @@ startup_script.path |          | path  | Relative path to a directory tree with
     startup_script:
       path: startup.sh
 ```
+###### Variants for Linux Terminal
+
+The allowed `linux_terminal` variants are:
+- it_cert [default]  (access to internet via shared Cloud NAT IP address)
```

Comment:
Nit: Maybe clarify inbound vs outbound internet access?

`outbound access to internet via shared Cloud NAT IP address, no inbound internet access`.

Reply:
how's that?

Reply:
LGTM